src/main/webapp/app/entities/participation-submission/participation-submission.component.html

```diff
@@ -0,0 +1,50 @@
+<h1>Submissions for Participation with id {{ participationId }}</h1>
+<br />
+<div class="table-responsive" *ngIf="submissions">
+    <ngx-datatable
+        id="participationSubmissionTable"
+        class="bootstrap"
+        [headerHeight]="50"
+        [limit]="20"
+        [columnMode]="'force'"
+        [footerHeight]="50"
+        [rowHeight]="'auto'"
+        [rows]="submissions"
+        [sorts]="[{ prop: 'id', dir: 'asc' }]"
+        [rowClass]=""
+    >
+        <ngx-datatable-column name="Id" prop="id">
+            <ng-template ngx-datatable-cell-template let-value="value">
+                {{ value }}
+            </ng-template>
+        </ngx-datatable-column>
+        <ngx-datatable-column name="Submission Type" prop="type">
+            <ng-template ngx-datatable-cell-template let-value="value">
+                {{ value }}
+            </ng-template>
+        </ngx-datatable-column>
+        <ngx-datatable-column name="Submission Date" prop="submissionDate">
+            <ng-template ngx-datatable-cell-template let-value="value">
+                {{ value }}
+            </ng-template>
+        </ngx-datatable-column>
+        <ngx-datatable-column name="Result" prop="result">
+            <ng-template ngx-datatable-cell-template let-value="value">
+                <jhi-result [result]="value"></jhi-result>
+            </ng-template>
+        </ngx-datatable-column>
+        <ngx-datatable-column name="Assessor" prop="result">
+            <ng-template ngx-datatable-cell-template let-value="value">
+                {{ value && value.assessor ? value.assessor.firstName : 'n.a' }}
```

Comment:
Please use the full name (firstName + lastName) and not only the first name.
We can provide a short hand on the client for this comparable to the getName() method in User.java

Reply:
Have a look into the json. It could be the case that `name` is already in there, then you simply need to add this as attribute into account.model.ts and it should be available automagically ;-)

Reply:
Both name and firstName give Fullname but I have replaced the firstName with name to avoid confusion.